bokehjs/test/integration/regressions.ts

```diff
@@ -4699,4 +4699,68 @@ describe("Bug", () => {
       ctx.drawImage(image, 0, 0, 200, 200)
     })
   })
+
+  describe("in issue #14549", () => {
+    it("doesn't prevent hover action upon bbox change", async () => {
+      function linspace(start: number, stop: number, num: number, endpoint = true) {
+        const div = endpoint ? (num - 1) : num
+        const step = (stop - start) / div
+        return Array.from({length: num}, (_, i) => start + step * i)
+      }
+
+      const n = 1000
+      const x = linspace(0, 20, n)
+      const y = x
+
+      const div = new Div({text: "some text"})
+      const source = new ColumnDataSource({data: {x, y}})
+
+      function hover_cb(_model: HoverTool, options: any) {
```

Comment:
`index` is always provided by the emitter:
```suggestion
      function hover_cb(_model: HoverTool, options: {index: Selection}) {
```